bot/cogs/pixelate.py

```diff
@@ -0,0 +1,43 @@
+import discord
+from discord import Color, Embed
+from PIL import Image
+from io import BytesIO
+from discord.ext import commands
```

Comment:
you should import `from discord.ext.commands` rather than importing `commands`, it would be much cleaner.